pkgs/tools/misc/yubikey-neo-manager/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, makeWrapper, python27Packages, pyside, pycrypto
+, libykneomgr, yubikey-personalization, libu2f-host }:
+
+python27Packages.buildPythonPackage rec {
+  namePrefix = "";
+  name = "yubikey-neo-manager-${version}";
+  version = "1.4.0";
+  src = fetchurl {
+    url = "https://developers.yubico.com/yubikey-neo-manager/Releases/${name}.tar.gz";
+    sha256 = "1isxvx27hk0avxwgwcwys2z8ickfs816ii1aklvmi09ak1rgrf1g";
+  };
+
+  propagatedBuildInputs = [ pyside pycrypto ];
+  patches = [ ./fix-pyside-requirement.diff ];
+
+  # XXX can we fix ctypes "load_library()" instead?
```

Comment:
It's difficult, as the ctypes stuff is inherently impure. Grep around for ctypes to see all kinds of hacks.

Wrapped binaries can be cheaply "fixed" with just LD_PRELOAD, but libraries as in #16313 are trickier.

I could open an issue to start a discussion, but if there was an easy fix it would likely have been implemented already. We can't tell ctypes itself up front about search paths as that would create a separate derivation for each consumer. And I don't see an easy way to monkey patch FFI libraries as they all implement it slightly differently.

I've updated the comment, as I hadn't really thought about it until now.